st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -343,7 +352,7 @@ def _run_action(self, action_node, parent_execution_id, params, wait_for_complet
             LOG.debug('%s: Task notify set to: %s', action_node.name, liveaction.notify)
 
         liveaction.context = {
-            'parent': str(parent_execution_id),
+            'parent': parent_context,
```

Comment:
I think so. But it would be good to at least test it (manually). Who knows what other thing might break as well :P